src/NuGet/Microsoft.CodeAnalysis.EditorFeatures.nuspec

```diff
@@ -10,14 +10,21 @@
 
       Supported Platforms:
       - .NET Framework 4.6
-      
+
       More details at https://aka.ms/roslyn-packages
 
       $commitPathMessage$
     </description>
     <dependencies>
       <dependency id="Microsoft.CodeAnalysis.Features" version="[$version$]" />
       <dependency id="Microsoft.CodeAnalysis.Elfie" version="$MicrosoftCodeAnalysisElfieVersion$" />
+      <dependency id="Microsoft.VisualStudio.Language.StandardClassification" version="$MicrosoftVisualStudioLanguageStandardClassificationVersion$" />
+      <dependency id="Microsoft.VisualStudio.Language.Intellisense" version="$MicrosoftVisualStudioLanguageIntellisenseVersion$" />
+      <dependency id="Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime" version="$MicrosoftVisualStudioImagingInterop140DesignTimeVersion$" />
+      <dependency id="Microsoft.VisualStudio.Text.UI" version="$MicrosoftVisualStudioTextUIVersion$" />
+      <dependency id="Microsoft.VisualStudio.Text.UI.Wpf" version="$MicrosoftVisualStudioTextUIWpfVersion$" />
```

Comment:
This shouldn't be here, since this layer isn't consuming UI.Wpf...right? Unless there's something else wrong?

Reply:
removed